docs/Release_notes.md

```diff
@@ -1,6 +1,30 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version 3.3.0, released January 31st, 2020
+--------------------------------------------------------
+
+**Summary:**
+CARMAPlatform release version 3.3.0 is comprised of two key enhancements. First the plugin manager is now able to return a list of services that match a user-requested capability. Second, there are new configurations for the Freightliner Cascadia, including lidar configurations which allow autoware to combine the two lidar devices' data into a single stream. 
+
+Enhancements in this release:
+- Issue 478: Capability Interface Implementation
+- Issue 489: Create configuration of carma-platform capable of operating on the freightliner cascadia using two lidars
+
```

Comment:
Need to add Issue 518 for the Garmin Repo. Just added this to the spreadsheet. John created the PR after our kickoff.

Reply:
Is 518 a request for hardware? We're only including software issues in the release notes.